src/utils/email_helper.ts

```diff
@@ -160,7 +166,9 @@ export const newMeetingEmail = async (
     },
     changeUrl: destinationAccountAddress
       ? `${appUrl}/dashboard/schedule?meetingId=${slot_id}&intent=${Intents.UPDATE_MEETING}`
-      : undefined,
+      : ownerAccountAddress
+      ? `${appUrl}/address/${ownerAccountAddress}?slot=${slot_id}`
```

Comment:
We have a util fotr getting the owners public address let's use that here instead.